src/components/PsAccounts.vue

```diff
@@ -1,130 +1,145 @@
 <template>
   <div id="psaccounts">
-    <BaseAlert
-      variant="danger"
-      :show="hasError"
-      dismissible
-      @dismissed="hasError = false"
-    >
-      <p class="acc-text-sm acc-leading-6">
-        {{ $t('psaccounts.accountManager.errorInstallEnable') }}
-      </p>
-    </BaseAlert>
-
-    <BaseAlert
-      v-if="validContext.errors && validContext.errors.length"
-      :class="{'acc-mt-4': hasError}"
-      variant="danger"
-    >
-      <p class="acc-text-sm acc-leading-6">
-        &lt;PsAccounts&gt; integration: Given context is invalid:
-        {{ validContext.errors.join(';') }}
-      </p>
-    </BaseAlert>
+    <AlertContextValidator
+      v-if="errors.length"
+      :errors="errors"
+      data-testid="account-context-validator-alert"
+    />
     <template v-else>
-      <PsAccountComponentAlertDisplay
+      <AlertModuleDependencies
+        :ps-accounts-is-enabled="context.psAccountsIsEnabled"
+        :ps-accounts-enable-link="context.psAccountsEnableLink"
+        :ps-accounts-is-installed="context.psAccountsIsInstalled"
+        :ps-accounts-install-link="context.psAccountsInstallLink"
+        :ps-accounts-is-uptodate="context.psAccountsIsUptodate"
+        :ps-accounts-update-link="context.psAccountsUpdateLink"
+        :ps-is17="context.psIs17"
+      />
+
+      <AlertShopUrlShouldExists
```

Comment:
I had never noticed, but here, the user can have up to 4 alerts at the same time? We could ask Guillaume to review this part in design for later?

Reply:
Yes but i think it's pretty uncommon to have the 4 alerts at the same time